docs/api/template/config.json

```diff
@@ -0,0 +1,28 @@
+{
+    "hideMemberFields": [
+        "TypeMeta"
+    ],
+    "hideTypePatterns": [
+        "ParseError$",
+        "List$"
+    ],
```

Comment:
If these were copied from gen-crd-api-reference-docs, consider sticking a README.md or something alongside these that explains what they are and where they came from (in case we want to fetch updated ones from upstream later we remember where we got them).